notebook_data_redirector/webhook_receiver.py

```diff
@@ -66,7 +76,18 @@ def lambda_handler(event, context):
             # clean up the relevant DynamoDB rows.
             return STATUS_SUCCESS
 
-        for file in common.iterate_files(folder):
+        folder_shared = common.is_box_file_public(folder)
```

Comment:
If it does turn out to be necessary to call `folder.get()` in iterate_files, then we'll need to do it here too

Reply:
didn't turn out to be necessary in iterate_files